testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +961,10 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+def test_CRYST1_Warning():
```

Comment:
To keep with the style of the other functions, I would suggest renaming this to a fully lowercase function name. Also, there are two warnings that could happen with the CRYST1 record, so I would suggest adding the word `em` in there somewhere.